src/main/resources/static/app/js/app.js

```diff
@@ -0,0 +1,17 @@
+(function () {
+    'use strict';
+
+    angular.module('booktrackrApp', ['ngRoute']);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["angular" is not defined.](https://www.codacy.com/app/martzrp/booktrackr/file/2613223913/issues/source?bid=3283479&fileBranchId=3311775#l4)